tests/main/snap-auto-import-asserts/task.yaml

```diff
@@ -0,0 +1,19 @@
+summary: Check that `snap auto-import` works as expected
+
+systems: [ubuntu-core-16-64]
```

Comment:
If possible it would be great to add here `ubuntu-core-16-arm-32` and `ubuntu-core-16-arm-64`